src/wx/cmdevents.cpp

```diff
@@ -2558,32 +2559,32 @@ EVT_HANDLER(GameBoyConfigure, "Game Boy options...")
 
 EVT_HANDLER(SetSize1x, "1x")
 {
-    config::OptDispScale()->SetDouble(1);
+    config::Proxy<config::OptionID::kDispScale>().Set(1);
```

Comment:
This is great, but I'd like some much simpler accessors for the main GUI code, this is a bit too long just to set an option.

Reply:
I added a macro and you can now treat the object more like a variable. The setter operator overloads still return `bool` rather than the object itself. Let me know if you'd like to change that so they're more variable-like. The `OptionProxy` object is still accessible directly for these needs.